README.md

```diff
@@ -361,7 +361,9 @@ mstorage.meta("<media_id>", "user", new CompletionHandler<MediaMeta>() {
 mstorage.meta("<media_id>", "user.<key>", new CompletionHandler<MediaMeta>() {
     @Override
     public void onCompleted(Map<String, String> userMeta) {
+        String value = userMeta.get("<key>");
         // Do something.
+        // "userMeta" may have only one element with key of "<key>".
```

Comment:
As read your comment, I thought it doesn't need explanation comment because sample code has added as example of getting value.
So I'll delete this comment.